### DISCUSSION TITLE
full asciidoc syntax support

### DISCUSSION THREAD

**Discussion by Phantasimay** [vim/vim#14431] [Ideas]
@aerostitich hi i have make enhancment for asciidoc syntax runtime. i was PR it to neovim because i use it on neovim. but i got recommended to PR here because  maintener of the syntax from here. how can i PR to vim if i make it on neovim repository?

**Comment by chrisbra** [comment]
You:
- clone the Vim repo
- you generate the patch using git (or download it from github) from your forked NeoVim Repo
- and apply the patch in your fork of the Vim repo
- create a PR of your changes to Vim Repo
